Initial post for 'SPM is usable now'

orta opened the pull request:
I needed to write #511 before I could write this, mainly because I though I would need to talk about the history in this post but I actually don't need to reference it at all. 

But the general pitch is: "Hey iOS folk, maybe it's time to use SPM in your projects now. Look at how it does dep man right, and your stuff is already working in that environment today."

mergesort commented on _posts/2019-01-05-its-time-to-use-spm.markdown:
```diff
@@ -0,0 +1,158 @@
+
+**Using Homebrew**
+
+Right now, a lot of folks use [homebrew][] to manage these types of dependencies. Homebrew is useful for rarely
```
Link to Homebrew is missing.

mergesort commented on _posts/2019-01-05-its-time-to-use-spm.markdown:
```diff
@@ -0,0 +1,158 @@
+**Using CocoaPods**
+
+You can hijack CocoaPod's dependency resolver, and locking system mixed with consistent dependency paths to handle
+your tools. This is better than using Homebrew, because everyone has the save version - and so you could reliably
```
Typo: same version, not save version.

mergesort commented on _posts/2019-01-05-its-time-to-use-spm.markdown:
```diff
@@ -0,0 +1,158 @@
+### Why Use SwiftPM?
+
+1. SPM works
+1. SPM can lock your dependencies correctly. :tada:
```
I assume this is meant to be 🎉 rather than `:tada:` spelled out?

mergesort commented on _posts/2019-01-05-its-time-to-use-spm.markdown:
```diff
@@ -0,0 +1,158 @@
+
+1. SPM works
+1. SPM can lock your dependencies correctly. :tada:
+1. The major ecosystem tools already support it, so you don't need to do any extra work
```
>The major ecosystem tools

I definitely understood the meaning and intent but that phrasing read a bit clunky to me. Maybe something like this would be better?

>The primary tools used in our ecosystem

orta commented:
Thanks @mergesort  - I agree with all of it, addressed the feedback and I'm switching the tada to a code highlight so it's more obvious that it's intentionally not the emoji

alloy commented on _posts/2019-01-05-its-time-to-use-spm.markdown:
```diff
@@ -0,0 +1,167 @@
+pushed the project over the finish line to being useful for iOS developers. `swift run` is contextually the same as
+`bundle exec` in that it will run a locally bundled version of your executable.
+
+This means you can run `run swift swiftlint` and reliably get the same results as your CI and fellow developers.
```
@orta Should this be `swift run swiftlint`?

orta replied:
lolyes
